### ISSUE TITLE
Modal: Input field in demo not programmatically connected to label

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#851]
### Expected behavior

Field input has explicit relationship with the label (e.g. using for and id attributes or wrapping input element in label). Showing correct coding practice for accessibility in demos. 
### Actual behavior

No explicit relationship between the elements.

**Comment by Blackbaud-PatrickOFriel** [comment]
In [v1.9.0](https://github.com/blackbaud/skyux/releases/tag/v1.9.0)
